client/data/help/use-help-search-query.ts

```diff
@@ -62,22 +63,30 @@ const fetchArticlesAPI = async (
 				path: `/help-center/articles?${ queryString }`,
 			} as APIFetchOptions ) ) as SearchResult[];
 		}
+
+		if ( articlesResponse?.length >= 0 ) {
+			searchResult = articlesResponse;
+		}
```

Comment:
Do we need this check? If we don't have any `articlesResponse` it will be `[]`, and it will get added in the `combinedResults`, not making any difference.